app/javascript/components/signups/SignUpForm.tsx

```diff
@@ -1,387 +1,393 @@
+                                    {homeOptions.map(({label, value}) => <option>{label}</option>)}
+                                </select>
+                            </Form.Item>
+
+                            <Divider>Animal Kind</Divider>
+                            <Form.Item name="animal_kind_preferences_attributes">
+                                <Checkbox.Group options={kindOptions}/>
+                            </Form.Item>
+
+                            <Divider>Gender Preference</Divider>
+                            <Form.Item name="animal_gender_preferences_attributes">
+                                <Checkbox.Group options={genderOptions}/>
+                            </Form.Item>
+
+                            <Divider>Age Preference</Divider>
+                            <Form.Item name="animal_age_preferences_attributes">
+                                <Checkbox.Group options={ageOptions}/>
+                            </Form.Item>
+
+                            <Divider>Size Preference</Divider>
+                            <Form.Item name="animal_size_preferences_attributes">
+                                <Checkbox.Group options={sizeOptions}/>
+                            </Form.Item>
+
+                            <Divider>Transportation</Divider>
+                            <Form.Item name="transportation">
+                                <select name="transportation">
+                                    <option value="access_to_car">Access to car</option>
+                                    <option value="car">Car</option>
+                                    <option value="no_car">No car</option>
+                                </select>
+                            </Form.Item>
+
+                            <Divider>Availability</Divider>
+                            <Row>
+                                <Col span={12}>
+                                    <Form.Item
+                                        name="available_from"
+                                        label="From"
+                                        rules={[
+                                            {
+                                                required: true,
+                                                message: "Please enter a start date for your availability.",
+                                            },
+                                        ]}
+                                    >
+                                        <DatePicker></DatePicker>
+                                    </Form.Item>
+                                </Col>
+
+                                <Col span={12}>
+                                    <Form.Item name="available_to" label="To (optional)">
+                                        <DatePicker></DatePicker>
+                                    </Form.Item>
+                                </Col>
+                            </Row>
+
+                            <Row>
```

Comment:
From here to line 375 is what I added. I formatted the file while working on it, which caused a lot of whitespace diff noise to appear.

Reply:
Love comments like these!

I usually like to create separate commits for styling changes so it's easy for reviewers to skip them.  Could ask for an easy approval or (for now) merge them into the `main` branch itself.